static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx

```diff
@@ -140,15 +140,20 @@ export function SpanGroupBreakdown({
             start && end
               ? {start: getUtcDateString(start), end: getUtcDateString(end), utc}
               : {statsPeriod: period};
-          if (['db', 'http'].includes(group['span.category'])) {
-            spansLinkQueryParams['span.module'] = group['span.category'];
+
+          if (group['span.category'] === 'Other') {
+            spansLinkQueryParams['!span.module'] = ['db', 'http'];
```

Comment:
This isn't a problem, but just wondering, why do you need this? The `!span.category` filter isn't enough?